addons/binding/org.openhab.binding.kodi/README.md

```diff
@@ -0,0 +1,64 @@
+# Kodi Binding
+
+This binding integrates the Kodi players (which used to be XBMC).
+
+## Introduction
+
+
+## Supported Things
+
+This binding supports only one thing: The Kodi player. All kodi devices are registered as an audio sink in the framework.
+
+
+## Discovery
+
+This binding can discover the Kodi players. 
+
+## Binding Configuration
+
+The binding can auto-discover the Kodi players present on your local network. The auto-discovery is enabled by default. To disable it, you can create a file in the services directory called Kodi.cfg with the following content:
```

Comment:
This line should actually be in the "Discovery" section. And no need to create a Kodi.cfg file; instead, the line should be added to conf/services/runtime.cfg.